persistence/src/main/scala/com/mesosphere/usi/persistence/Repository.scala

```diff
@@ -0,0 +1,50 @@
+package com.mesosphere.usi.persistence
+
+import akka.{Done, NotUsed}
+import akka.stream.scaladsl.Source
+import com.mesosphere.usi.core.models.{PodRecord, ReservationRecord}
+
+import scala.concurrent.Future
+
+
+/**
+  * Base trait for storing records which are used by the Scheduler.
```

Comment:
Should we link to the concept of `records`?

Reply:
Sounds good, but we need to write it first.